tools/license-builder.sh

```diff
@@ -33,7 +33,7 @@ addlicense "c-ares" "deps/cares" \
            "$(sed -e '/^ \*\/$/,$d' -e '/^$/d' -e 's/^[/ ]\* *//' ${rootdir}/deps/cares/src/ares_init.c)"
 addlicense "HTTP Parser" "deps/http_parser" "$(cat deps/http_parser/LICENSE-MIT)"
 addlicense "ICU" "deps/icu" \
-           "$(sed -e '1,/COPYRIGHT AND PERMISSION NOTICE/d' -e '/^<hr/,$d' -e 's/^<\/*p>$//' ${rootdir}/deps/icu/license.html)"
+           "$(sed -e '1,/ICU License - ICU 1\.8\.1 and later/d' -e :a -e 's/<[^>]*>//g;/</N;//ba' ${rootdir}/deps/icu/license.html)"
```

Comment:
can you regex that exact version number out of this so we don't have to update this tool with each upgrade?